packages/react/src/components/UIShell/SwitcherItem.tsx

```diff
@@ -58,6 +58,10 @@ interface BaseSwitcherItemProps {
    * Specify where to open the link.
    */
   target?: HTMLAttributeAnchorTarget;
+  /**
+   * The rel property for the link.
+   */
+  rel?: string;
```

Comment:
Is there a specific type off of HTMLAttributes or DOMAttributes or something that we could use here? It's not any string - it's one of a specific set of allowed/expected strings, right?

[MDN's list](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) is "some of the most important existing keywords" 🤔 very possible this doesn't exist but figured I'd ask